appengine-java11/tasks/src/main/java/com/example/task/CreateTask.java

```diff
@@ -0,0 +1,167 @@
+      .longOpt("project-id")
+      .desc("The Google Cloud Project, if not set as GOOGLE_CLOUD_PROJECT env var.")
+      .hasArg()
+      .argName("project-id")
+      .type(String.class)
+      .build();
+
+  private static Option QUEUE_OPTION = Option.builder("q")
+      .required()
+      .longOpt("queue")
+      .desc("The Cloud Tasks queue.")
+      .hasArg()
+      .argName("queue")
+      .type(String.class)
+      .build();
+
+  private static Option LOCATION_OPTION = Option.builder("l")
+      .required()
+      .longOpt("location")
+      .desc("The region in which your queue is running.")
+      .hasArg()
+      .argName("location")
+      .type(String.class)
+      .build();
+
+  private static Option PAYLOAD_OPTION = Option.builder("p")
+      .longOpt("payload")
+      .desc("The payload string for the task.")
+      .hasArg()
+      .argName("payload")
+      .type(String.class)
+      .build();
+
+  private static Option IN_SECONDS_OPTION = Option.builder("s")
+      .longOpt("in-seconds")
+      .desc("Schedule time for the task to create.")
+      .hasArg()
+      .argName("in-seconds")
+      .type(int.class)
+      .build();
+
+  public static void main(String... args) throws Exception {
+    Options options = new Options();
+    options.addOption(PROJECT_ID_OPTION);
+    options.addOption(QUEUE_OPTION);
+    options.addOption(LOCATION_OPTION);
+    options.addOption(PAYLOAD_OPTION);
+    options.addOption(IN_SECONDS_OPTION);
+
+    if (args.length == 0) {
+      printUsage(options);
+      return;
+    }
+
+    CommandLineParser parser = new DefaultParser();
+    CommandLine params = null;
+    try {
+      params = parser.parse(options, args);
+    } catch (ParseException e) {
+      System.err.println("Invalid command line: " + e.getMessage());
+      printUsage(options);
+      return;
+    }
+
+    String projectId;
+    if (params.hasOption("project-id")) {
+      projectId = params.getOptionValue("project-id");
+    } else {
+      projectId = System.getenv(GOOGLE_CLOUD_PROJECT_KEY);
+    }
+    if (projectId.equals(null) || projectId.equals("")) {
+      printUsage(options);
+      return;
+    }
+
+    String queueName = params.getOptionValue(QUEUE_OPTION.getOpt());
+    String location = params.getOptionValue(LOCATION_OPTION.getOpt());
+    String payload = params.getOptionValue(PAYLOAD_OPTION.getOpt(), "default payload");
+
+    // Instantiates a client.
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+
+      // Variables provided by the CLI.
+      // projectId = "my-project-id";
+      // queueName = "my-appengine-queue";
+      // location = "us-central1";
+      // payload = "hello";
+
+      // Construct the fully qualified queue name.
+      String queuePath = QueueName.of(projectId, location, queueName).toString();
+
+      // Construct the task body.
+      Task.Builder taskBuilder = Task
+          .newBuilder()
+          .setAppEngineHttpRequest(AppEngineHttpRequest.newBuilder()
```

Comment:
This line break looks like it might be a style violation (checkstyle should catch it when you get it added)

Reply:
done.

Reply:
Still seems wrong - see "[Where to Line Break" ](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break)